core/jsoncollector.class.inc.php

```diff
@@ -141,12 +141,13 @@ public function Prepare()
 			} else {
 				$aDataGet = [];
 			}
-			$iSynchroTimeout = (int)Utils::GetConfigurationValue('itop_synchro_timeout', 600); // timeout in seconds, for a synchro to run
-			$aCurlOptions = Utils::GetCurlOptions($iSynchroTimeout);
+			
+			$aCurlOptions = static::GetCurlOptions();
```

Comment:
Cannot be called static
```suggestion
			$aCurlOptions = $this->GetCurlOptions();
```